server/core/src/https/v1.rs

```diff
@@ -2908,8 +2908,37 @@ pub async fn debug_ipinfo(
     Ok(Json::from(vec![ip_addr]))
 }
 
+#[utoipa::path(
+    get,
+    path = "/v1/jwk/{key_id}",
+    responses(
+        (status=200, body=Jwk, content_type="application/json"),
+        ApiResponseWithout200,
+    ),
+    security(("token_jwt" = [])),
+    tag = "v1/jwk",
+    operation_id = "public_jwk_key_id_get"
+)]
+pub async fn public_jwk_key_id_get(
+    State(state): State<ServerState>,
+    Path(key_id): Path<String>,
```

Comment:
Are key_id's always UUIDs or are they identifiers from the keys themselves? I tried to dig in but couldn't really follow it that deep... 😄

Reply:
KeyIDs are strings that are provided from the keysource itself. For a TPM it will be the object "name" which is generated by the tpm, and in this case from compact-jwt it's a truncated 256 of the public key. It's a "unique id" of the key.